src/battle_util.c

```diff
@@ -7834,6 +7834,13 @@ bool32 CanMegaEvolve(u8 battlerId)
     u8 partnerPosition = GetBattlerPosition(BATTLE_PARTNER(battlerId));
     struct MegaEvolutionData *mega = &(((struct ChooseMoveStruct*)(&gBattleResources->bufferA[gActiveBattler][4]))->mega);
 
+#ifdef ITEM_EXPANSION
```

Comment:
Similar to the Z Move PR, it might be better to change this to
```suggestion
#ifdef ITEM_MEGA_BRACELET
```

Reply:
Why? :eyes:

Reply:
So they can just add the bracelet without the full expansion. Just a thought though, not really necessary xD

Reply:
Thing is, a user could want to add some other type of keystone item. The Mega Bracelet isn't the only one, so #ifdef'fing under an assumption like that feels a bit eh to me, personally.

Reply:
True.